consensus-types/wrapper/beacon_block.go

```diff
@@ -145,6 +149,115 @@ func BuildSignedBeaconBlock(blk interfaces.BeaconBlock, signature []byte) (inter
 	}
 }
 
+// BuildSignedBeaconBlockFromExecutionPayload takes a signed, blinded beacon block and converts into
+// a full, signed beacon block by specifying an execution payload.
+func BuildSignedBeaconBlockFromExecutionPayload(
+	blk interfaces.SignedBeaconBlock, payload *enginev1.ExecutionPayload,
```

Comment:
Something to consider is to separate out `interfaces.SignedBeaconBlock` with `interfaces.SignedBlindBeaconBlock`. Nice improvement for later